client/src/pages/SignUp/SignUp.tsx

```diff
@@ -1,15 +1,15 @@
 import CssBaseline from '@material-ui/core/CssBaseline';
-import Paper from '@material-ui/core/Paper';
-import Box from '@material-ui/core/Box';
 import Grid from '@material-ui/core/Grid';
 import { FormikHelpers } from 'formik';
 import Typography from '@material-ui/core/Typography';
 import useStyles from './useStyles';
 import register from '../../helpers/APICalls/register';
-import SignUpForm from './SignUpForm/SignUpForm';
-import AuthHeader from '../../components/AuthHeader/AuthHeader';
 import { useAuth } from '../../context/useAuthContext';
 import { useSnackBar } from '../../context/useSnackbarContext';
+import { Link } from 'react-router-dom';
+import { Divider } from '@material-ui/core';
+import SignUpForm from './SignUpForm/SignUpForm';
+import loginDemoUser from '../../helpers/APICalls/loginDemoUser';
```

Comment:
Are these duplicated from #48?

Let's adjust the base of the branch to that one so we see relevant changes.

Reply:
done!